Application/EdFi.Admin.DataAccess/Repositories/ClientAppRepo.cs

```diff
@@ -230,7 +230,6 @@ public ApiClient GetClient(string key)
                 return context.Clients.Include(c => c.Application)
                     .ThenInclude(c => c.Vendor)
                     .ThenInclude(c => c.VendorNamespacePrefixes)
-                    .Include(c => c.Application.Profiles)
```

Comment:
That line did not compile, but I should not have permanently removed it. We need to add this back with the appropriate `ThenInclude` statement.